src/Enzo/enzo_EnzoMethodPmDeposit.cpp

```diff
@@ -2,6 +2,7 @@
 
 /// @file     enzo_EnzoMethodPmDeposit.cpp
 /// @author   James Bordner (jobordner@ucsd.edu)
+/// Edited:   Stefan Arridge (stefan.arridge@gmail.com)
```

Comment:
```suggestion
/// @author:   Stefan Arridge (stefan.arridge@gmail.com)
```
I would encourage you to use `@author` here since that's a keyword recognized by Doxygen.